catalog/measure/catalog-measure-api/src/main/java/ddf/measure/Distance.java

```diff
@@ -124,10 +125,52 @@ public double getAs(LinearUnit unitOfMeasure) {
     }
 
     /**
-     * The Enum LinearUnit for the distance in meters
+     * Enumeration for the linear units supported by DDF.
      */
     public enum LinearUnit {
-        METER, KILOMETER, NAUTICAL_MILE, MILE, FOOT_U_S, YARD
-    }
+        METER, KILOMETER, NAUTICAL_MILE, MILE, FOOT_U_S, YARD;
+
+        static Map<String, LinearUnit> stringsToValues = new HashMap<>();
```

Comment:
You could store the alternate mappings as values of their respective enum instances (via constructor), and have fromString() loop through .values() checking both primary and alternate values for a match. Does that work/make sense?

Or you could just have a map with multiple keys mapping to the same enum value. ;) That would reduce some of this code as well.